components/tools/OmeroWeb/omeroweb/webclient/views.py

```diff
@@ -1216,27 +1216,14 @@ def load_data(request, o1_type=None, o1_id=None, o2_type=None, o2_id=None,
             if index == 0:
                 index = fields[0]
 
-        # We don't know what our menu is so we're setting it to None.
-        # Should only raise an exception below if we've been asked to
-        # show some tags which we don't care about anyway in this
-        # context.
-        show = Show(conn, request, None)
-        # Constructor does no loading.  Show.first_selected must be
-        # called first in order to set up our initial state correctly.
-        try:
-            first_selected = show.first_selected
-            if first_selected is not None:
-                wells_to_select = list()
-                paths = show.initially_open + show.initially_select
-                for path in paths:
-                    m = Show.PATH_REGEX.match(path)
-                    if m is None:
-                        continue
-                    if m.group('object_type') == 'well':
-                        wells_to_select.append(m.group('value'))
-                context['select_wells'] = ','.join(wells_to_select)
-        except IncorrectMenuError:
-            pass
+        # Show parameter will be well-1|well-2
+        show = request.REQUEST.get('show')
```

Comment:
Possibly although using `grep` will not scale as you need to maintain a list of deprecated features/bugs. Browsing quickly, it might be possible long-term to pass a `-Werror` flag to `python manage.py test` which will fail on deprecation errors. This is not critical and might be easier to implement if we re-architecture the components but worth keeping in mind.